### DISCUSSION TITLE
a folder called `__MACOSX` is present in plugins folder

### DISCUSSION THREAD

**Discussion by isAAAc** [kanboard/kanboard#5206] [Installation & Configuration]
### Checklist

- [x] I verified that Kanboard is correctly installed
- [ ] I checked that the problem does not come from a plugin
- [x] I checked that the problem is not already reported

### Actual behaviour

a folder called `__MACOSX` is present in plugins folder, this is sending many notification in `/var/log/apache2/kanboard.octopuce.fr_error.log` until i delete it.
### Expected behaviour

no folder == no notification, 
i'm on debian, so no OSX stuff should be there
### Screenshots

![Screenshot_2021-08-19_16-11-22](https://user-images.githubusercontent.com/1662903/130084204-bb018079-bee6-4445-9841-12e79884ed5e.png)
### Logs
```
[Thu Aug 19 16:08:41.066819 2021] [php7:notice] [pid 309593] [client 2001:67c:288:2::233:34986] [critical] __MACOSX: Unable to load this plugin class: \\Kanboard\\Plugin\\__MACOSX\\Plugin
```

### Configuration

- Kanboard version: a fresh v1.2.20
- Database type and version: Sqlite3
- PHP version: 7.4
- OS: Debian 10.10
- Browser: N/A

**Comment by fguillot** [comment]
How did you installed Kanboard? The archive doesn't contains this folder. The [archive](https://github.com/kanboard/kanboard/archive/refs/tags/v1.2.20.zip) is directly coming from GitHub.

https://github.com/kanboard/kanboard/tree/master/plugins
https://github.com/kanboard/kanboard/releases/tag/v1.2.20

This folder could be coming from one of the plugin you installed. The plugin author could have generated the archive manually on its Mac.

**Comment by felag** [comment]
Hello,

**Edit: it came from an update yesterday. From a bug on greenwing plugin/theme : removing theme solved issue. Sorry for the noise.**

I have same problem this morning (all working fine yesterday, ~~AFAIK no installing/upgrading action taken~~).

On my side, `__MACOSX` folder is related to `greenwing`:
```bash
bash-5.1# ls -l plugins/
total 0
drwxr-xr-x    1 nginx    nginx          230 Aug 22  2019 Calendar
drwxr-xr-x    1 nginx    nginx          112 Aug 22  2019 CardPushDate
drwxr-xr-x    1 nginx    nginx          188 Jun 19  2020 Gantt
drwxr-xr-x    1 nginx    nginx          190 Aug 22  2019 GitlabAuth
drwxr-xr-x    1 nginx    nginx          224 Aug 22  2019 GitlabWebhook
drwxr-xr-x    1 nginx    nginx          380 May 13 15:47 Greenwing
drwxrwxr-x    1 nginx    nginx          222 Mar 13  2020 Group_assign
drwxr-xr-x    1 nginx    nginx          216 Aug 22  2019 Mattermost
drwxr-xr-x    1 nginx    nginx          270 May 13 15:48 MetaMagik
drwxr-xr-x    1 nginx    nginx           18 May 13 15:47 __MACOSX
bash-5.1# ls -l plugins/__MACOSX/
total 0
drwxr-xr-x    1 nginx    nginx          400 May 13 15:47 Greenwing
```

@isAAAc By any chance, can you see any subfolder(s) in your `__MACOSX` directory ?
(from you input, it seems related to `MarkdownPlus` plugin)


More generally speaking is there a process (cron, admin action, upstream version check or other) that may create folders inside `plugins` directory ?
